crates/autopilot/src/solvable_orders.rs

```diff
@@ -269,7 +269,16 @@ impl SolvableOrdersCache {
         counter.record(&auction.orders);
 
         if self.store_in_db {
-            let _id = self.database.replace_current_auction(&auction).await?;
+            let auction_id = self.database.replace_current_auction(&auction).await?;
+            self.database
+                .insert_auction_prices(auction_id, &auction.prices)
```

Comment:
If we do it this way, then we need to clean up the `auction_prices` table for auctions without settlements.

Reply:
I think the discussion about that was had and conclusion was that it wasn't substantially "large" enough to bother.

Reply:
I disagree, right now, that is around 200 prices every 10s. We've had 845 in the past 24 hours, so assuming the numbers stay roughly the same, that is `24 * 60 * 60 / 10 - 845` auctions with 200 prices, or around 1559000 rows per day.

I think this is too much.

Reply:
That is DEFINITELY too much. Apologies for the confusion.

Reply:
Fixed here: https://github.com/cowprotocol/services/pull/1282